### DISCUSSION TITLE
License problem, no one reply us for 3 weeks !!!

### DISCUSSION THREAD

**Discussion by Linksoft2007** [airsdk/Adobe-Runtime-Support#595] [Q&A]
We have looking for help for 3 weeks, no one reply the mail.
We write the mails to these three mail every day, but there's NO ONE REPLY !!!
adobe.support@harman.com'; 'Adobe Support' <adobe.support@alerts.harman.com>; 'Andrew.frost@Harman.com'

Please help to fix the license, the client purchase the license in May, but it invalid today.
Here is the error message.
ADT license file for developer reported as: INVALID
License file has been removed: please contact adobe.support@harman.com to renew your license

Thank you, looking forward to your reply soon.

**Comment by ajwfrost** [comment]
Hi
I've just searched my emails, I do see one email from you on Friday .. apologies for missing that, I'll get one of the DB team to look into this asap..
thanks

**Comment by ajwfrost** [comment]
Hi

Having looked into this .. unless I'm getting very mistaken with emails, you should have support for this, we had another email about it earlier on Friday and had added this back in. I've just tried and it's working.

Please can you confirm by typing into a command line:
adt -license
and let us know what it outputs?

Also to double-check, your license Developer ID starts with "c64"? I'm trying to see any link between your email and the one that was sent earlier, these appear to be coming from two different companies but having the same developer ID ..?

thanks

**Comment by ladigitale** [comment]
Hi, we have this kind of problem with our license.
We bought the license 6 months ago and cannot use it.
No solution provided despite many mails.
ADT license is reported as invalid.
Should we open a new ticket, or keep this one?

For information our Developer ID does not start with c64.

Thank you,
I hope to hear from you soon.

**Comment by ajwfrost** [comment]
Okay I think I have the email now, I had confused your thread here with a different thread in our support inbox... so, I will check with the team and get this sorted out..

thanks

**Comment by ladigitale** [comment]
OK, but that's not my thread.
I am writing today in this thread which already exists because the problem seems very close to mine.

**Comment by ajwfrost** [comment]
Ah that would explain it.. so I hope (!) that the original one was already sorted; your one (assuming your dev id starts b5c) is currently being checked..
thanks

**Comment by ladigitale** [comment]
Thank you.
It does start with b5c.

**Comment by ladigitale** [comment]
Hello,

I am very worried, I must publish my apps.
Do you have any news regarding my licence checking

**Comment by ajwfrost** [comment]
@ladigitale we'll contact you off-line about this one..
